webapp/channels/src/components/tabs/tabs/index.tsx

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import classNames from 'classnames';
+import React from 'react';
+import {Tabs as ReactBootstrapTabs} from 'react-bootstrap';
+import type {SelectCallback} from 'react-bootstrap';
+
+import './style.scss';
+
+type Props = {
+    children?: React.ReactNode;
+    id?: string;
+    activeKey?: any;
+    mountOnEnter?: boolean | undefined;
+    unmountOnExit?: boolean | undefined;
+    onSelect?: SelectCallback | undefined;
```

Comment:
Is `| undefined` needed in this cases? Doesn't the `?` cover for that?

Reply:
Fixed. I probably blindly copied this from the underlying component